psydac/core/bsplines.py

```diff
@@ -58,11 +60,14 @@ def find_span( knots, degree, x ):
         Knot span index.
 
     """
+
+
     # Knot index at left/right boundary
     low  = degree
     high = len(knots)-1-degree
 
     # Check if point is exactly on left/right boundary, or outside domain
+
```

Comment:
See above. Please remove

Reply:
Fixed in https://github.com/pyccel/psydac/commit/c9eac22256b58bb6b2138192ad3cc5766ae2d143